CHANGELOG.md

```diff
@@ -1,4 +1,78 @@
-# dbt_apple_store_source version.version
+# dbt_apple_store_source v0.5.0
+[PR #18](https://github.com/fivetran/dbt_apple_store_source/pull/18) includes the following updates:
+
+## Breaking Changes: Schema Change
+- Following the connector's [Nov 2024 Update](https://fivetran.com/docs/connectors/applications/apple-app-store/changelog#november2024) to sync from the [App Store Connect API](https://developer.apple.com/documentation/appstoreconnectapi), we've updated this dbt package to reflect the new schema which includes the following changes:
+
+- *Models Removed*:
+  - `stg_apple_store__app_store_device`
```

Comment:
I think `stg_apple_store__crashes_platform_version` got removed too?

Reply:
added to changelog!